recipe/meta.yaml

```diff
@@ -44,8 +46,6 @@ requirements:
     - python
     - py-lief
     - pyyaml
-    - ripgrep
```

Comment:
Why remove this?

Reply:
hm I was under the impression that it wasn't used in conda-build but I see now that it's optionally used, I'll restore it